_spinnaker/terraform_integration.md

```diff
@@ -146,8 +147,11 @@ Armory ships the following versions of the Terraform binary as part of the Terra
 
 **Note**: Terraform binaries are verified by checksum and with Hashicorp's GPG key before being installed into our release.
 
-To use Terraform, you must indicate to the Terraformer microservice
-the path to the binary within the microservice to use:
+To use the Terraform Integration, you must specify the path to the Terraform binary you want to use. This path also includes the Terraform version. This version is used for the **SYSTEM_DEFINED** version value when a user creates a Terraform Integration stage in Deck. This version is not used if the user selects a different supported version from the menu.
```

Comment:
"you must specify the path to the Terraform binary you want to use" -- I don't think this is true? You can also just use the dropdown to select your version and then you don't need that blurb below in `terraformer-local.yml`. We might want to be more clear about there being two ways to set the version and why someone might want to use `SYSTEM_DEFINED`.

Reply:
Got it:

* Be explicit that this only needs to be set if they want to configure a value for SYSTEM_DEFINED.
* Benefits of using SYSTEM_DEFINED, e.g. how you need the same Terraform version across your whole pipeline for some workflows and SYSTEM_DEFINED is an easy way to make sure of that.

Any other benefits?